### PR TITLE
[RUN-3915][RUN-3813] Add page for JDK installation instructions (Mx10.11)

### PR DESCRIPTION
**Pull request by atennapel**
Moved to 10.11

### CODE REVIEW

File: content/en/docs/refguide/installation/install.md [hunk 1]
```diff
@@ -65,7 +65,7 @@ Sometimes you can run into problems when installing Studio Pro. One work-around
 The prerequisites are the following:
 
 * [Microsoft .NET Desktop Runtime 6.0.x](https://dotnet.microsoft.com/en-us/download/dotnet/6.0) – Mendix recommends using version 6.0.6 or above
-* [Eclipse Temurin JDK 11 (x64)](https://github.com/adoptium/temurin11-binaries/releases)
+* [Eclipse Temurin JDK 11, 17 or 21 (x64)](https://adoptium.net/temurin/releases/?version=21) (see [JDK Installation](refguide/jdk-installation) if you want to install another version of the JDK)
```
Comment (MarkvanMents) [inline content/en/docs/refguide/installation/install.md]: We seem to use the names Eclipse, Temurin, and Adoptium interchangeably.

I think we should stick to one and use it throughout. Would you prefer
* Adoptium
* Eclipse Temurin
* Temurin
* Adoptium Temurin

Or something else?

**Review by MarkvanMents — COMMENTED** [review]
Hi Albert (@atennapel)
Thanks for the PR. I've reviewed it and made a couple of updates.
I also have a comment about the name we use for the JDK. Can you have a look and let me know what you think.
Thanks.

File: content/en/docs/refguide/installation/install.md (hunk 1 above)
Comment (atennapel) [reply content/en/docs/refguide/installation/install.md]: Agreed. I think "Eclipse Temurin" is the most accurate when referring to the JDK. As far as I understand "Eclipse Temurin" is the name of the product (the JDK) while "Adoptium" is the name of the group that develops this product. From the site: "Eclipse Temurin is the name of the OpenJDK distribution from Adoptium."

Comment (MarkvanMents) [reply content/en/docs/refguide/installation/install.md]: I unified the naming in this commit: https://github.com/mendix/docs/pull/7585/commits/88b4c2b61738a86b5b4cd82ac5ddba5849a7fe29

File: content/en/docs/refguide/installation/system-requirements.md [hunk 2]
```diff
@@ -201,7 +201,7 @@ The Mendix Docker buildpack supports the following Kubernetes versions:
 
 ### 6.3 Java {#java}
 
-When running Mendix on a server, you will need Java Runtime Environment (JRE) 11 or 17. To download an OpenJDK distribution from Adoptium, see [Adoptium Installation](https://adoptium.net/temurin/releases). To download a commercial Oracle distribution, see [Java SE Downloads](https://www.oracle.com/technetwork/java/javase/downloads/index.html).
+When running Mendix on a server, you will need Java Runtime Environment (JRE) 11 or 17. To download an Eclipse Temurin OpenJDK distribution from Adoptium, see [Eclipse Temurin™ Latest Releases](https://adoptium.net/temurin/releases). To download a commercial Oracle distribution, see [Java SE Downloads](https://www.oracle.com/technetwork/java/javase/downloads/index.html).
 
 {{% alert color="warning" %}}
 Compatibility with JDK 17 has been released with Studio Pro version 10.8. Compatibility with Java 21 is planned for Studio Pro version 10.10, to be released in April 2024. Mendix recommends switching to a Studio Pro version compatible with Java 21 once it is available.
```
Comment (MarkvanMents) [inline content/en/docs/refguide/installation/system-requirements.md]: I guess this warning needs changing too? Will you do this as part of this PR, or is it being done somewhere else?

**Review by MarkvanMents — COMMENTED** [review]
I've unified the naming in the docs.
I've also spotted another reference to JDK 11 and 17 which doesn't include 21. Will this be picked up in a different PR, or should it be changed here?

File: content/en/docs/refguide/installation/system-requirements.md (hunk 2 above)
Comment (atennapel) [reply content/en/docs/refguide/installation/system-requirements.md]: Indeed, that should be part of this PR.

**Comment by atennapel** [comment]
@MarkvanMents This has been moved to 10.11

**Comment by MarkvanMents** [comment]
Thanks for the update @atennapel - I've changed the label.

File: content/en/docs/refguide/installation/system-requirements.md (hunk 2 above)
Comment (atennapel) [reply content/en/docs/refguide/installation/system-requirements.md]: Done.

File: content/en/docs/refguide/installation/system-requirements.md [hunk 3]
```diff
@@ -42,7 +42,7 @@ The following frameworks are required. They will be installed automatically by t
 
 * Microsoft .NET 6.0.x desktop runtime (x64) and all applicable Windows security patches
 * Microsoft Visual C++ 2019 Redistributable Package (x64)
-* A Java Developer Kit (JDK) version 11 or 17 - if not yet installed on your machine, Mendix will install 'Eclipse Temurin JDK 11 (x64)'
+* A Java Developer Kit (JDK) version 11, 17 or 21 - if not yet installed on your machine, Mendix will install 'Eclipse Temurin JDK 21 (x64)'
```
Comment (MarkvanMents) [inline content/en/docs/refguide/installation/system-requirements.md]:
```suggestion
* A Java Developer Kit (JDK) version 11, 17, or 21 - if not yet installed on your machine, Mendix will install 'Eclipse Temurin JDK 21 (x64)'
```

File: content/en/docs/refguide/installation/system-requirements.md [hunk 4]
```diff
@@ -204,10 +204,10 @@ The Mendix Docker buildpack supports the following Kubernetes versions:
 
 ### 6.3 Java {#java}
 
-When running Mendix on a server, you will need Java Runtime Environment (JRE) 11 or 17. To download an OpenJDK distribution from Adoptium, see [Adoptium Installation](https://adoptium.net/temurin/releases). To download a commercial Oracle distribution, see [Java SE Downloads](https://www.oracle.com/technetwork/java/javase/downloads/index.html).
+When running Mendix on a server, you will need Java Runtime Environment (JRE) 11, 17 or 21. To download an Eclipse Temurin OpenJDK distribution from Adoptium, see [Eclipse Temurin™ Latest Releases](https://adoptium.net/temurin/releases). To download a commercial Oracle distribution, see [Java SE Downloads](https://www.oracle.com/technetwork/java/javase/downloads/index.html).
```
Comment (MarkvanMents) [inline content/en/docs/refguide/installation/system-requirements.md]:
```suggestion
When running Mendix on a server, you will need Java Runtime Environment (JRE) 11, 17, or 21. To download an Eclipse Temurin OpenJDK distribution from Adoptium, see [Eclipse Temurin™ Latest Releases](https://adoptium.net/temurin/releases). To download a commercial Oracle distribution, see [Java SE Downloads](https://www.oracle.com/technetwork/java/javase/downloads/index.html).
```

**Review by MarkvanMents — APPROVED** [review]
Looks good - approving.

File: content/en/docs/refguide/installation/install.md [hunk 5]
```diff
@@ -56,8 +56,8 @@ If you run into problems installing Studio Pro, one work-around is to restart yo
 
 The prerequisites are the following:
 
-* [Microsoft .NET Desktop Runtime 6.0.x](https://dotnet.microsoft.com/en-us/download/dotnet/6.0) – Mendix recommends using version 6.0.6 or above.
-* [Eclipse Temurin JDK 11 (x64)](https://github.com/adoptium/temurin11-binaries/releases)
+* [Microsoft .NET Desktop Runtime 6.0.x](https://dotnet.microsoft.com/en-us/download/dotnet/6.0) – Mendix recommends using version 6.0.6 or above
+* [Eclipse Temurin JDK 11, 17 or 21 (x64)](https://adoptium.net/temurin/releases/?version=21) (see [JDK Installation](refguide/jdk-installation) if you want to install another version of the JDK)
```
Comment (MarkvanMents) [inline content/en/docs/refguide/installation/install.md]:
```suggestion
* [Eclipse Temurin JDK 11, 17, or 21 (x64)](https://adoptium.net/temurin/releases/?version=21) (see [JDK Installation](/refguide/jdk-installation) if you want to install another version of the JDK)
```

**Review by jveldhuizen — APPROVED** [review]
Looks good - approved on behalf of Runtime Core.

**Pull request merged**